test/passing/js_source.ml.opts

```diff
@@ -1,2 +1,3 @@
 --max-iters=3
 --profile=janestreet
+--impl
```

Comment:
I think this should be generated by `gen.ml` for every `.ocp` rules.

Reply:
not necessary if we don't invoke ocamlformat after, removed.